examples/compiled/rect_heatmap.vg.json

```diff
@@ -179,11 +179,6 @@
         },
         "axisY": {
             "minExtent": 30
-        },
```

Comment:
What is the difference between ramp and heatmap? Is one for marks that fill the space and one for marks that may overlap and leave holes? If so, how is blues better than viridis?